tablelog.py

```diff
@@ -0,0 +1,133 @@
+import argparse
+import copy
+import glob
+import json
+import logging
+import os
+import random
+import shutil
+import time
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import numpy as np
+import pytest
+import torch
+import tqdm
+import wandb
+from torch.utils.data import DataLoader
+
+
+from utils import elmo_sentence_decode
+
+class TableLog():
+    """A class to log W&B tables for specific examples over each epoch of retrofitting."""
+    
+    def __init__(self, num_table_examples:int, train_dataloader: DataLoader, test_dataloader: DataLoader, columns:List[str]):
```

Comment:
nit: put a space between colon and type, like `num_table_examples: int` not `num_table_examples:int`